### ISSUE TITLE
Installation Error

### ISSUE THREAD

**Issue by cyberboysumanjay** [Nekmo/telegram-upload#5]
* telegram-upload version: 0.1.3
* Python version: 3.5
* Operating System: Elementary OS Linux

### Description
I tried to install telegram-upload using command 
`sudo -H pip3 install telegram-upload` 
I got this error

> Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-z3lzlhr5/telegram-upload/

Describe what you were trying to get done.
telegram-upload installed successfully.

Tell us what happened, what went wrong, and what you expected to happen.
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-z3lzlhr5/telegram-upload/

### What I Did
Tried sudo -H pip install telegram-upload

**Comment by Nekmo** [comment]
```
(tmp-52566d013519094) [nekmo@homura ~/.virtualenvs/tmp-52566d013519094]$ pip3 install telegram-upload
Collecting telegram-upload
  Downloading https://files.pythonhosted.org/packages/c8/00/6e6cbf5b8966980fd0cba8e3337a39559b32ea80ccbe80e4c91ef632781d/telegram-upload-0.1.3.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>                                                                                                                                                                                                         
      File "/tmp/pip-install-yhxu3o29/telegram-upload/setup.py", line 10, in <module>                                                                                                                                                              
        from pip.req import parse_requirements                                                                                                                                                                                                     
    ImportError: No module named 'pip.req'                                                                                                                                                                                                         
                                                                                                                                                                                                                                                   
    ----------------------------------------                                                                                                                                                                                                       
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-yhxu3o29/telegram-upload/                                                                                                                                          
```
Confirmed.

**Comment by Nekmo** [comment]
Solved in release 0.1.5
